.travis.yml

```diff
@@ -14,8 +14,10 @@ matrix:
     - php: 7.0
   fast_finish: true
 
-before_script:
+before_install:
   - composer self-update
```

Comment:
@GrahamCampbell 

It's actually quite useful if you want to make use of latest features - which may not be available on Travis yet, see for example #158.

Alternatively, we could check in `composer.phar`.